pybossa/view/projects.py

```diff
@@ -1737,6 +1737,10 @@ def tasks_browse(short_name, page=1, records_per_page=None):
             args["regular_user"] = regular_user
             # default page size for worker view is 100
             per_page = records_per_page if records_per_page in allowed_records_per_page else task_list_default_records_per_page
+        elif view_type == 'tasklist' and n_available_tasks_for_user(project, current_user.id) == 0:
+            # When no tasks are available, redirect to browse all tasks view.
+            new_path = '/project/{}/tasks/browse'.format(project.short_name)
```

Comment:
Maybe add something like 
```
flash("No tasks available")
```

Reply:
Good suggestion, thank you.

The flash wording showing for workers might be a little confusing. For project owners, they would simply be directed to the browse tasks page. Note, the behavior of this has not changed except that project owners will now see the browse tasks page instead of a forbidden error.

![no-tasks-avail](https://github.com/user-attachments/assets/90df4bd3-b41f-449e-a019-61af1f62a8ae)